content/en/docs/architecture/ci-operator.md

```diff
@@ -270,6 +268,26 @@ images:
   to: test-image
 ```
 
+## How Builds Work In CI
+Every container image built in CI is implemented by
+[a native OpenShift build](https://docs.openshift.com/container-platform/4.12/cicd/builds/understanding-image-builds.html), i.e.,
+`ci-operator` uses the parameters in the `images` stanza from the configuration file to define `builds` options on CI clusters.
+
+
+| `images`              | `build`                                        |
+|-----------------------|------------------------------------------------|
+| `.build_args`         | `.spec.strategy.dockerStrategy.buildArgs`      |
+| `.dockerfile_literal` | `.spec.source.dockerfile`                      |
+| `.dockerfile_path`    | `.spec.strategy.dockerStrategy.dockerfilePath` |
+| `.context_dir`        | `.spec.source.contextDir`                 |
```

Comment:
`images` are built using the `Image` source type, so this is not correct.  See:

- https://github.com/openshift/ci-tools/blob/d50fac661e3c70527167ccb506f45b03b79df682/pkg/steps/project_image.go#L40-L105
- https://github.com/openshift/ci-tools/blob/d50fac661e3c70527167ccb506f45b03b79df682/pkg/steps/source.go#L288

We do use `spec.source.contextDir` in cases where other build types are used, but not here.